src/App.tsx

```diff
@@ -175,7 +175,7 @@ function getMessage(logging: JSON): undefined | getMessageReturn {
               `${formattedTimestamp}`
             );
             source.push(
-              `${logs[msg]["message"]["source"]}`
+              `Application: ${logs[msg]["message"]["application_name"]} \n${logs[msg]["message"]["source"]}`
```

Comment:
I think application name and source should be two separate columns, perhaps source should also have a more descriptive name such as "host"?